.github/workflows/on-push-run-build.yml

```diff
@@ -0,0 +1,42 @@
+name: on-push-run-build
+on:
+  push:
```

Comment:
Something like `on: [push, pull_request]` would be needed to handle PRs.

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events

Checking this I think you're right. 

I didn't realise appveyor was there because I saw there were already github actions for tagging and doco. 

But this also allows other contributors to enable actions on their fork and test their work too.